internal/test/wmcb/wmcb_test.go

```diff
@@ -112,10 +112,14 @@ func TestWMCB(t *testing.T) {
 	for _, vm := range framework.WinVMs {
 		log.Printf("Testing VM: %s", vm.GetCredentials().InstanceId())
 		wVM := &wmcbVM{vm}
-		files := strings.Split(*filesToBeTransferred, ",")
-		for _, file := range files {
-			err := wVM.CopyFile(file, remoteDir)
-			require.NoError(t, err, "error copying %s to the Windows VM", file)
+		folders := strings.Split(*dirsToBeTransferred, ",")
+		// all the files in `/payload` directory will be transferred to `C:\Temp`
+		// for files not in `/payload`, new directory will be created in `C:\Temp` which resembles the linux folder path
+		for _, folder := range folders {
```

Comment:
Please use `directories` or `dirs` instead of `folder` everywhere including comments. In comments please use `directories`.